docs/source/tutorial_optimization.md

```diff
@@ -777,6 +776,88 @@ assert opt_along_path_minimization_data['enrichments'] == 9
 assert opt_along_path_adaptively_minimization_data['enrichments'] == 4
 ```
 
+## Adaptive trust-region optimization
+
+Moreover, we can use trust-region methods to extend the adaptive enrichment with
+respect to the underlying error estimator.
```

Comment:
Although this is true for the TR method, it is also already true for the method that has been discussed in the former section. I would rather tend to remove the former method and maybe even the one before (see the main comment of the review).